lib/js/jscore.nim

```diff
@@ -110,3 +110,15 @@ proc parse*(l: JsonLib, s: cstring): JsRoot {.importcpp.}
 since (1, 5):
   func debugger*() {.importjs: "debugger@".}
     ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger
+
+  template jsexport*(symbol: auto; alias = "") =
```

Comment:
Why `auto` instead of `untyped` or `typed`?

Reply:
wouldn't `typed` be better than untyped though?
untyped affects overload resolution, and is primarely useful when something can't be typed (eg called iterators pending https://github.com/nim-lang/Nim/pull/17196, or other specific instances, but doesn't seem like it applies here)
(in private helpers, it's ok though, but this is a public API)